instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -84,20 +129,13 @@ defmodule OpentelemetryEcto do
     time_unit = Keyword.get(config, :time_unit, :microsecond)
     additional_attributes = Keyword.get(config, :additional_attributes, %{})
 
-    db_type =
-      case type do
-        :ecto_sql_query -> :sql
-        _ -> type
-      end
-
     # TODO: need connection information to complete the required attributes
     # net.peer.name or net.peer.ip and net.peer.port
     base_attributes = %{
-      "db.type": db_type,
-      source: source,
-      "db.instance": database,
+      "ecto.db.adapter": to_string(adapter),
+      "db.system": db_system(config[:db_system], adapter),
       "db.name": database,
-      "db.url": url,
+      "db.sql.table": source,
```

Comment:
We don't actually know that an Ecto source is a sql table, do we here?

Reply:
https://hexdocs.pm/ecto/Ecto.Schema.Metadata.html#module-source

The doc says it will be a table or a collection, I don't know if we have a way to determine if it's a table, but it could be a table. I'm not sure if for SQL adapters it will be always a table.

Reply:
I figure for a sql adapter it will alway be a table. So thats all we need to know to know to include it (I think).

Reply:
It could be a view, for example, not sure if it would break your semantics.

Reply:
I don't think it's breaks the semantics here, view is like a virtual table.